feflow/tests/test_lambdaprotocol.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pytest
+
+from feflow.utils import lambda_protocol
+
+running_on_github_actions = os.environ.get('GITHUB_ACTIONS', None) == 'true'
+
+
```

Comment:
Because the prtocool is a bit different (i.e. it takes `windows`) we probably should move these tests over too: https://github.com/OpenFreeEnergy/openfe/blob/afdffb1c9fa77e5df3b08851e917e7f66ff97a13/openfe/tests/protocols/test_openmm_equil_rfe_protocols.py#L878-L887